packages/headless/src/features/did-you-mean/did-you-mean-actions.ts

```diff
@@ -8,6 +8,14 @@ export const enableDidYouMean = createAction('didYouMean/enable');
 
 export const disableDidYouMean = createAction('didYouMean/disable');
 
+export const disableAutomaticQueryCorrection = createAction(
+  'didYouMean/correction/disable'
```

Comment:
The identifier could be a bit more verbose, to explain the fact that it's controlling the `automatic` part of the correction.